docs/data/upload.md

```diff
@@ -0,0 +1,113 @@
+---
+title: "Upload Data"
+linkTitle: "Upload Data"
+description: "Upload data to the Viam app from your local computer or mobile device using the data client API, Viam CLI, or Viam mobile app."
+weight: 40
+type: "docs"
+tags: ["data management", "cloud", "sync"]
+image: "/services/icons/data-capture.svg"
+imageAlt: "Upload data to the Viam app"
+images: ["/services/icons/data-capture.svg"]
+# SME: Alexa Greenberg
+---
+
+If you configured [data capture](/data/capture/) on your machine, data is automatically uploaded to the Viam cloud from the directory and at the interval you specified.
```

Comment:
Nit: I see what you're going for, but `from the directory and at the interval you specified` comes across a little awkward until resolved at the every end (with qualifier `you specified`). What about a slight clarity tweak:

```suggestion
When you configure [data capture](/data/capture/) on your machine, data is automatically [synced to the Viam cloud](/data/cloud-sync/) from the directory you specified and at the interval you set in the configuration.
```

`If` vs `when` is my preference, absolutely disregard if you disagree!